package.json

```diff
@@ -26,12 +26,12 @@
     "README.md",
     "README_zh.md",
     "index.ts",
-    "index.d.ts"
+    "exceljs.d.ts"
   ],
   "scripts": {
     "test": "npm run test:full",
     "test:es5": "export EXCEL_BUILD=es5 && npm run test:full",
-    "test:full": "npm run build && npm run test:unit && npm run test:integration && npm run test:end-to-end && npm run test:jasmine",
+    "test:full": "npm run build && npm run test:unit && npm run test:integration && npm run test:end-to-end && npm run test:jasmine && npm run test:typescript",
```

Comment:
Added TS tests to `test:full`, which should now work as they no longer contain a failing test case.